gdk/src/wayland/wayland_monitor.rs

```diff
@@ -0,0 +1,10 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+glib::wrapper! {
+    #[doc(alias = "GdkWaylandMonitor")]
+    pub struct WaylandMonitor(Object<ffi::GdkWaylandMonitor>) @extends crate::Monitor;
+
```

Comment:
Could implement `gdk_wayland_monitor_get_wl_output`

Reply:
I only implemented methods returning gobject types. I prefer not to bind functions returning pure wayland types as it would mean that we need to bind them. Or I could just declare a type alias to `c_void` and call it a day?

Reply:
Yes, and you use Wayland-rs crate to provide safe wrapper for the Wayland types

Reply:
Just to be sure we agree: we don't add dependency on wayland crates and I bind the methods by making them unsafe and returning the pointer. If users want to use it, they can use wayland wrappers to safely handle the returned pointer.

Did I understand you correctly?

Reply:
Well, wayland-client already provides safe bindings wrappers for wayland types returned by gdk-wayland. So you would make the ffi functions return a c_void and provide safe wrappers by using wayland-client. That's what i did in gdk4-wayland anyway